frontend/package.json

```diff
@@ -7,13 +7,15 @@
         "dev": "vite",
         "build": "npx tailwindcss -i ./src/input.css -o ./src/App.css; vite build",
         "watch": "npx tailwindcss -i ./src/input.css -o ./src/App.css --watch",
+        "tail": "npx tailwindcss -i ./src/input.css -o ./src/App.css",
```

Comment:
Will we need this ? As, this command is already added with the build command.
If it is still needed then please rename to "tailwind" instead of just "tail", the "tail" is a bit confusing with the `tail` command in Linux (I myself thought that it will do some logging kinda stuff when I only looked only at the name).

Reply:
not required, will remove in a commit in future.

Reply:
Merging now only as the merge is needed for further work progress, please cover this comment in next commit.